src/System.Private.ServiceModel/tools/scripts/WCFCustomCleanup.cmd

```diff
@@ -0,0 +1,36 @@
+@if "%_echo%" neq "on" echo off
```

Comment:
I admittedly haven't run this script on my box yet, but I did notice that one dependency as mentioned above. I do agree with Ron's comment though that running this command should mean that it'll start the WCF service without fail except under exceptional circumstances. 

I ran into similar pitfalls with Linux certificate installers and the like where I depended on, but didn't explicitly state or check for, such dependencies. It makes the experience a bit more fluid and nicer for the dev